### PR TITLE
Add Python packages pcdsdevices, pcdswidgets

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc @hhslepicka

### CODE REVIEW

**Comment by hhslepicka** [comment]
I accept to be a maintainer of the packages here.

**Comment by hhslepicka** [comment]
@conda-forge/help-python this recipe is ready to go...

**Comment by isuruf** [comment]
Please fix the linter hints at https://github.com/conda-forge/staged-recipes/pull/11122#issuecomment-603552878

File: recipes/pcdsdevices/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2018, The Board of Trustees of the Leland Stanford Junior
```
Comment (isuruf) [inline recipes/pcdsdevices/LICENSE.md]: Can you delete this file and use the one from the source tarball?

Comment (klauer) [reply recipes/pcdsdevices/LICENSE.md]: The latest release of the library does not yet include the license, though that will be fixed in the next release.

**Comment by hhslepicka** [comment]
@isuruf about the bot comment, in many other recipes we use `SLAC Open` for the license type and it never was a problem. Is this a new check on the linter? What should we use since it is a SLAC/Stanford specific license?

File: recipes/pcdsdevices/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - pcdsdevices
+
+about:
+  home: https://github.com/pcdshub/pcdsdevices
+  doc_url: https://pcdshub.github.io/pcdsdevices/
+  dev_url: https://github.com/pcdshub/pcdsdevices
+  license: SLAC Open
```
Comment (isuruf) [inline recipes/pcdsdevices/meta.yaml]:
```suggestion
  license: LicenseRef-SLAC
```

Comment (hhslepicka) [reply recipes/pcdsdevices/meta.yaml]: The linter didn't like this change. Can we switch back to `SLAC Open` and proceed?

Comment (isuruf) [reply recipes/pcdsdevices/meta.yaml]: Can you open an issue in conda-smithy repo?

Comment (hhslepicka) [reply recipes/pcdsdevices/meta.yaml]: Done https://github.com/conda-forge/conda-smithy/issues/1272

File: recipes/pcdswidgets/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - pcdswidgets
+
+about:
+  home: https://github.com/pcdshub/pcdswidgets
+  doc_url: https://pcdshub.github.io/pcdswidgets/
+  license: LicenseRef-SLAC
+  license_family: Other
```
Comment (isuruf) [inline recipes/pcdswidgets/meta.yaml]:
```suggestion
  license_family: BSD
```

Comment (hhslepicka) [reply recipes/pcdswidgets/meta.yaml]: The guideline for `Other` came from this review here: https://github.com/conda-forge/staged-recipes/pull/9639/files#r327228549
Should we also change it at our other packages later?

File: recipes/pcdsdevices/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipes/pcdsdevices/meta.yaml]: This license is BSD-3-Clause-LBNL right? https://spdx.org/licenses/BSD-3-Clause-LBNL.html

Comment (hhslepicka) [reply recipes/pcdsdevices/meta.yaml]: No... https://github.com/pcdshub/pcdsdevices/blob/master/LICENSE.md

File: recipes/pcdswidgets/meta.yaml (hunk 3 above)
Comment (isuruf) [reply recipes/pcdswidgets/meta.yaml]: Yes

File: recipes/pcdsdevices/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipes/pcdsdevices/meta.yaml]: Only the copyright changed here right? See section 2.1.3 of https://spdx.org/spdx-license-list/matching-guidelines

cc @scopatz 

Btw, the SPDX recommendation by the linter is new.

Comment (hhslepicka) [reply recipes/pcdsdevices/meta.yaml]: @isuruf at a first glance it looks like it is just the copyright. Should we add in this case just BSD and move on?

Comment (isuruf) [reply recipes/pcdsdevices/meta.yaml]: I think we should go with either `BSD-3-Clause-LBNL` or `LicenseRef-BSD-3-Clause-SLAC`. If the former, there's no warnings. If the latter, there'll be a warning, but we can ignore that and merge and fix the linter later.

Comment (hhslepicka) [reply recipes/pcdsdevices/meta.yaml]: Let's do ` LicenseRef-BSD-3-Clause-SLAC`. 👍

Comment (isuruf) [reply recipes/pcdsdevices/meta.yaml]: I also opened https://github.com/spdx/license-list-XML/issues/994

**Comment by klauer** [comment]
Thanks a lot for the help getting the license stuff sorted, @isuruf!

Is there anything else I need to do here?

**Pull request merged**